vhdbuilder/packer/packer_source.sh

```diff
@@ -318,19 +318,19 @@ copyPackerFiles() {
   cpAndMode $SNAPSHOT_UPDATE_SERVICE_SRC $SNAPSHOT_UPDATE_SERVICE_DEST 644
   cpAndMode $SNAPSHOT_UPDATE_TIMER_SRC $SNAPSHOT_UPDATE_TIMER_DEST 644
 
-  if [[ $OS != $MARINER_OS_NAME ]]; then
+  if [[ $OS != $MARINER_OS_NAME ]] && [[ $OS != $AZURELINUX_OS_NAME ]]; then
     cpAndMode $DOCKER_CLEAR_MOUNT_PROPAGATION_FLAGS_SRC $DOCKER_CLEAR_MOUNT_PROPAGATION_FLAGS_DEST 644
     cpAndMode $NVIDIA_MODPROBE_SERVICE_SRC $NVIDIA_MODPROBE_SERVICE_DEST 644
     cpAndMode $PAM_D_COMMON_AUTH_SRC $PAM_D_COMMON_AUTH_DEST 644
     cpAndMode $PAM_D_COMMON_PASSWORD_SRC $PAM_D_COMMON_PASSWORD_DEST 644
     cpAndMode $USU_SH_SRC $USU_SH_DEST 544
   fi
-  if [[ $OS == $MARINER_OS_NAME ]]; then
+  if [[ $OS == $MARINER_OS_NAME ]] || [[ $OS == $AZURELINUX_OS_NAME ]]; then
     cpAndMode $CONTAINERD_SERVICE_SRC $CONTAINERD_SERVICE_DEST 644
     cpAndMode $MPU_SH_SRC $MPU_SH_DEST 544
 
-    # MarinerV2 uses system-auth and system-password instead of common-auth and common-password.
-    if [[ ${OS_VERSION} == "2.0" ]]; then
+    # Mariner/AzureLinux uses system-auth and system-password instead of common-auth and common-password.
+    if [[ ${OS_VERSION} == "2.0" ]] || [[ ${OS_VERSION} == "3.0" ]]; then
```

Comment:
this is a bit confusing since there's now 2 ways to distinguish V3 - `"$OS_VERSION" == 3.0 and $OS == AZURELINUX_OS_NAME`, I'd recommend using one of the aforementioned methods in locations where both are available to reduce confusion

Reply:
Changed to using $OS == AZURELINUX_OS_NAME to keep it consistent - [commit](https://github.com/Azure/AgentBaker/pull/4808/commits/0f0c0202c3e88a067b2f0acc6a3f025cb8b6f5da)